src/pages/Home/sections/LiqiuditySection/components/PoolCard.tsx

```diff
@@ -0,0 +1,66 @@
+import { Trans } from "@lingui/macro";
```

Comment:
looks like too much nesting

We usually try to keep flat structure 

Probably just put it in src/pages/Home would be enough or even in src/components